epersongroups.md

```diff
@@ -0,0 +1,390 @@
+# EPerson groups Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**GET /api/eperson/groups**
+
+## Single EPerson Group
+**GET /api/eperson/groups/<:uuid>**
+
+```json
+{
+  "id": "617cf46b-535c-42d5-9d22-327ce2eff6dc",
+  "uuid": "617cf46b-535c-42d5-9d22-327ce2eff6dc",
+  "name": "Administrator",
+  "handle": null,
+  "metadata": {
+    "dc.title": [
+      {
+        "value": "Administrator",
+        "language": null,
+        "authority": "",
+        "confidence": -1,
+        "place": 0
+      }
+    ]
+  },
+  "permanent": true,
+  "type": "group",
+  "_links": {
+    "subgroups": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/groups"
+    },
+    "parentgroups": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/parentgroups"
+    },
+    "epersons": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/epersons"
+    },
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc"
+    }
+  }
+}
+```
+
```

Comment:
it would be useful to add a sentence about the permanent attribute, i.e. it is a system required group that cannot be removed.

Moreover, we should also decide if the administrator is allowed or not to patch this information (or any metadata of permanent groups) and note that the DELETE of a group flagged as permanent should fail

Reply:
I've included this description, it's easy to understand